migrations/2020_01_14_130500_change_permission_groups_add_is_hidden.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+use Flarum\Database\Migration;
+
+return Migration::addColumns('groups', [
+    'is_hidden' => ['boolean', 'default' => 0]
```

Comment:
```suggestion
    'is_hidden' => ['boolean', 'default' => false]
```

At least that's how we do it elsewhere. :wink: